SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -415,11 +415,16 @@ Process {
 FunctionDef {
 	var raw1, raw2, <code, <selectors, <constants, <prototypeFrame, <context, <argNames, <varNames;
 	var <sourceCode;
+	var <filenameSymbol;
+	var <charPos;
+	var <lineNumber;
 
 	// a FunctionDef is defined by a code within curly braces {}
 	// When you use a FunctionDef in your code it gets pushed on the stack
 	// as an instance of Function
 
+	filename { ^filenameSymbol !? (_.asString) }
```

Comment:
should better be:

filename { ^filenameSymbol !? { filenameSymbol.asString } }